WordPress/Classes/ViewRelated/Stats/Today Widgets/iOS 14 Widgets/BlogListViewController+StatsWidgets.swift

```diff
@@ -0,0 +1,48 @@
+import WidgetKit
+
+extension BlogListViewController {
+
+    @objc func refreshStatsWidgetsSiteList() {
+
+        guard let currentData = HomeWidgetTodayData.read() else {
+            return
+        }
+        let blogService = BlogService(managedObjectContext: ContextManager.shared.mainContext)
+        let updatedSiteList = blogService.visibleBlogsForWPComAccounts()
+
+        let newData = updatedSiteList.reduce(into: [Int: HomeWidgetTodayData]()) { result, element in
```

Comment:
This is very nitpicky, as the code is good as it is.  I was just thinking that replacing `element` with something like `site` could make the code slightly more readable.

Obviously, feel free to disregard if you disagree or simply prefer the current term.

Reply:
totally agree, changed!!